src/app/qgsdxfexportdialog.cpp

```diff
@@ -800,6 +808,239 @@ void QgsDxfExportDialog::deselectDataDefinedBlocks()
 }
 
 
+void QgsDxfExportDialog::loadSettingsFromFile()
+{
+  QgsSettings settings;
+  const QString lastUsedDir = settings.value( QStringLiteral( "dxf/lastSettingsDir" ), QDir::homePath() ).toString();
+
+  const QString fileName = QFileDialog::getOpenFileName( this, tr( "Load DXF Export settings" ), lastUsedDir,
+                           tr( "XML file" ) + " (*.xml)" );
+  if ( fileName.isNull() )
+  {
+    return;
+  }
+
+  bool resultFlag = false;
+
+  QDomDocument myDocument( QStringLiteral( "qgis" ) );
+
+  // location of problem associated with errorMsg
+  int line, column;
+  QString myErrorMessage;
+
+  QFile myFile( fileName );
```

Comment:
I think generally speaking, QGIS code base is trying to avoid using the `my` prefix for variable names. I've seen a bunch introduced here, I think it should be removed (e.g. file instead of myFile). Consider it valid for all such instances in this PR :)

Reply:
Good to know, thanks. (It was copy&pasted from somewhere else from the QGIS source code)

Reply:
@gacarrillor , yeah, we have quite a few of these lingering around still. We just have to insure we're not adding new ones ;)

Reply:
Let's do that.

Reply:
Addressed in https://github.com/gacarrillor/QGIS/pull/37/commits/b8c6bd37a2121acc90ffdabd77feeb2163ff8001